<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VirusTotal File Checker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_Main">
    <item>
     <widget class="QTableView" name="tableViewResults">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Controls">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_ApiKey">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>API Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditApiKey">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSetApiKey">
          <property name="text">
           <string>Save Key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelApiKeyStatus">
          <property name="text">
           <string>Key not set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Search">
        <item>
          <widget class="QLabel" name="labelSearchPrompt">
           <property name="text">
            <string>Search (Name/Hash):</string>
           </property>
          </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearch">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter name or hash...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSearch">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonClearSearch">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_FileActions">
        <item>
         <widget class="QPushButton" name="pushButtonAddFiles">
          <property name="text">
           <string>Add Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonScanSelected">
          <property name="text">
           <string>Scan Selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
